plaid/institutions.go

```diff
@@ -13,6 +13,32 @@ type Institution struct {
 	Name         string       `json:"name"`
 	Products     []string     `json:"products"`
 	CountryCodes []string     `json:"country_codes"`
+
+	// Included when `options.include_status` is true.
+	InstitutionStatus InstitutionStatus `json:"status"`
+
+	// Included when `options.include_optional_metadata` is true.
+	PrimaryColor string `json:"primary_color"`
+	// Included when `options.include_optional_metadata` is true.
+	URL string `json:"url"`
+	// Included when `options.include_optional_metadata` is true.
+	Logo string `json:"logo"`
+}
+
+type InstitutionStatus struct {
+	ItemLogins ItemLogins `json:"item_logins"`
+}
+
+type ItemLogins struct {
+	Status           string                     `json:"status"`
+	LastStatusChange string                     `json:"last_status_change"`
```

Comment:
should this be `time.Time` instead of a string? not sure how we serialize other timestamps.

Reply:
:zap: